<?xml version="1.0"?>
<!-- launches DOL node only -->
<launch>

    <!--dol Params-->
    <arg name="mobileGoalTopic" default="/move_base_simple/goal"/> 
    <arg name="loop_rate_value" default="10"/>
    <arg name="goal_dist_tolerance" default = "0.1"/> <!-- in meters -->
    <arg name="rel_motion_mode" default = "false"/>
    <arg name="map_frame" default="map"/>
    <arg name="robot_frame" default="base_link"/>
    <arg name="dol_params" default="$(find mw_maxl_planner)/config/dol_config"/>

    <!-- =============== DOL =============== -->
    <node pkg="mw_maxl_planner" name="mw_maxl_dol_node" type="mw_maxl_dol_node" output="screen">
        <rosparam command="load" file="$(arg dol_params)/mw_maxl_planner.yaml" subst_value="true"/>
        <param name="mobileGoalTopic" value="$(arg mobileGoalTopic)"/>
        <param name="dist_tolerance" value="$(arg goal_dist_tolerance)"/> 
        <param name="loop_rate_value" value="$(arg loop_rate_value)"/>
        <param name="rel_motion_mode" value="$(arg rel_motion_mode)"/>
        <param name="map_frame" value="$(arg map_frame)"/>
        <param name="robot_frame" value="$(arg robot_frame)"/>
    </node>
</launch>    